demos/embeddings/README.md

```diff
@@ -1,39 +1,35 @@
 
-The notebooks demonstrate the following algorithms.
-- `stellargraph-node2vec.ipynb` The **Node2Vec** algorithm [1] for representation learning on homogeneous graphs.
-- `stellargraph-metapath2vec.ipynb` The **Metapath2Vec** algorithm [2] for representation learning on heterogeneous graphs.
-- `embeddings-unsupervised-graphsage-cora.ipynb` The **Unsupervised GraphSAGE** algorithm [5] for representation learning on homogeneous graphs with node features.
-- `stellargraph-attri2vec-citeseer.ipynb` The **attri2vec** algorithm [6] for representation learning on the homogeneous graph with node features.
+Node representations can also be computed from (semi-)supervised models, using the output of a hidden layer as the embedding vector for nodes or edges. StellarGraph provides some [demonstrations of node classification][nc] and [link prediction][lp], some of which include computing and visualising node or edge embeddings.
 
-All examples demonstrate how to calculate embedding vectors for a graph's nodes in just a few lines of Python code.
-The learned node representations can be used in numerous downstream tasks such as node attribute inference, link
-prediction, and community detection.
+[nc]: ../node-classification/README.md
+[lp]: ../link-prediction/README.md
+[cd]: ../community_detection/README.md
 
+## Find algorithms and demos for a graph
 
-## References
+This table lists all representation learning demos, including the algorithms trained, how they are trained, the types of graph used, and the tasks demonstrated.
 
-**1.** Node2Vec: Scalable Feature Learning for Networks. A. Grover, J. Leskovec. ACM SIGKDD International Conference
-on Knowledge Discovery and Data Mining (KDD), 2016. ([link](https://snap.stanford.edu/node2vec/))
+| demo | algorithm(s) | training method | node features | downstream tasks shown |
+|---|---|---|---|---|
+| [Deep Graph Infomax][dgi] | GCN, GAT, PPNP, APPNP, GraphSAGE | `DeepGraphInfomax` (mutual information) | yes | visualisation, node classification |
+| [Unsupervised GraphSAGE][graphsage] | GraphSAGE | `UnsupervisedSampler` (link prediction) | yes | visualisation, node classification |
+| [Attri2Vec][attri2vec] | Attri2Vec | `UnsupervisedSampler` (link prediction) | yes | visualisation |
+| [Metapath2Vec][metapath2vec] | Metapath2Vec | natively unsupervised | | visualisation |
+| [Node2Vec][node2vec] | Node2Vec | natively unsupervised | | visualisation |
+| [Watch Your Step][wys] | Watch Your Step | natively unsupervised | | visualisation, node classification |
```

Comment:
Should the node features column contain "no" instead of being empty?

```suggestion
| [Watch Your Step][wys] | Watch Your Step | natively unsupervised | no | visualisation, node classification |
```

Reply:
I thought that it was easier to read/scan the tables with whitespace for negatives, instead of `no`. For instance, for a table with more columns like link prediction:

| Demo | Algorithm | Node features | Heterogeneous | Temporal |
|---|---|---|---|---|
| [GCN][gcn] | GCN | yes | no | no |
| [Attri2Vec][attri2vec] | Attri2Vec | yes |no | no |
| [GraphSAGE][graphsage] | GraphSAGE | yes |no | no |
| [HinSAGE][hinsage] | HinSAGE | yes | yes | no |
| [Node2Vec][node2vec] | Node2Vec | no | no | no|
| [CTDNE][ctdne] | CTDNE | no | no | yes |
| [ComplEx][complex] | ComplEx | no | yes, multiple edge types | no |
| [DistMult][distmult] | DistMult | no | yes, multiple edge types | no |

[gcn]: gcn/cora-gcn-links-example.ipynb
[attri2vec]: attri2vec/stellargraph-attri2vec-DBLP.ipynb
[graphsage]: graphsage/cora-links-example.ipynb
[hinsage]: hinsage/movielens-recommender.ipynb
[node2vec]: random-walks/cora-lp-demo.ipynb
[ctdne]: random-walks/ctdne-link-prediction.ipynb
[complex]: knowledge-graphs/complex.ipynb
[distmult]: knowledge-graphs/distmult.ipynb

I find that much noisier, and, e.g. the `yes` for heterogeneous is harder to see at a glance.

The same is true for this one, but it is less bad, since there's only one columns with a few blanks. However, I think consistency is good and so leave no/negatives out.

Reply:
oh yeah I see what you mean - the blanks are way easier to processs 👍